Packs/Gurucul/Playbooks/playbook-GraCase.yml

```diff
@@ -81,10 +47,10 @@ tasks:
       brand: ""
     nexttasks:
       '#none#':
-      - "3"
+      - "2"
     scriptarguments:
       value:
-        simple: ${Gra.User.Anomalies.anomaly_name}
+        simple: ${incident.graanomalyname}${incident.graanomalies}
```

Comment:
this seems wrong, what is the value you are trying to retrieve here ?

Reply:
Here anomaly name or multiple anomaly name column value will print as per selection of "Fetch Incident Cases" drop-down in configuration.